GA/function_list.txt

```diff
@@ -0,0 +1,48 @@
+This File will contain the list of functions we need for our GA to opperate
+
+
+DataRead:
+Read in generational data and fitness scores to be used.
+
+DataWrite
+Write out generational data and fitness scores
+
+GenerateARA
+Random generate an invididual for the ARA antenna design
+
+GeneratePUEO
+Random generate an individual for the PUEO antenna design
+
+ConstraintARA
+Check the antenna constraints of an ARA individual 
+
+Constraint PUEO
+Check the antenna constraints of a PUEO individual
+
+Sort
+Sort the inviduals by fitness score ranking
+
+Selection:
+Calls selection methods based on proportion and then places the individual's locations into a vector
+probably shuffle the array so that selection methods don't dominate opperators
```

Comment:
```suggestion
Calls selection methods based on proportion and then places the individual's locations into a vector
probably shuffle the array so that selection methods don't dominate operators
```